server/odc-service/src/main/java/com/oceanbase/odc/service/flow/model/CreateFlowInstanceReq.java

```diff
@@ -50,6 +50,11 @@
 
 @Data
 public class CreateFlowInstanceReq {
+    /**
+     * FlowInstanceId
+     */
+    @JsonProperty(access = Access.READ_ONLY)
+    private Long id;
```

Comment:
Why do we need an `id` for the creating operation?

Reply:
This field is used to transfer flow instance id to fill in template variables.